test/engineSpec.js

```diff
@@ -1505,14 +1469,18 @@ describe('Engine', function() {
 
     describe('isMetroAreaOnRespondentPanel', function() {
```

Comment:
Given that `isMetroAreaOnRespondentPanel` went from ~5 lines with no branches to ~50 lines with a bunch of branches, I think we could use more than one 'happy' path test.